src/core/server/metrics/metrics_service.ts

```diff
@@ -84,4 +95,20 @@ export class MetricsService
     }
     this.metrics$.complete();
   }
+
+  private setupOpsMetricsEventType(
+    analytics: AnalyticsServiceSetup,
+    metricsObservable: Observable<OpsMetrics>,
+    analyticsMetricsLogger: Logger
+  ) {
+    analytics.registerEventType<OpsMetrics>({
+      eventType: 'core-ops_metrics',
+      schema: opsMetricsSchema,
+    });
+    metricsObservable.subscribe((metrics: OpsMetrics) => {
+      const { message } = getEcsOpsMetricsLog(metrics);
+      analyticsMetricsLogger.info(`reporting opsMetrics: ${message}`);
```

Comment:
do we really need to log the metrics we report to telemetry?

Reply:
We don't need to log the metrics we're reporting, since the logs will essentially be duplicates of those we're already logging from the metrics service, albeit at a different log level.

I've removed the logger.